x-pack/plugins/ml/server/routes/schemas/modules.ts

```diff
@@ -138,8 +138,8 @@ const moduleSchema = schema.object({
   type: schema.string(),
   logo: schema.maybe(schema.any()),
   logoFile: schema.maybe(schema.string()),
-  defaultIndexPattern: schema.string(),
-  query: schema.any(),
+  defaultIndexPattern: schema.maybe(schema.string()),
+  query: schema.maybe(schema.any()),
   jobs: schema.arrayOf(schema.any()),
   datafeeds: schema.arrayOf(schema.any()),
   kibana: schema.any(),
```

Comment:
I believe kibana can also be missing so should be wrapped in `schema.maybe(...`

Reply:
Updated in https://github.com/elastic/kibana/pull/191633/commits/e0913cac6feabab3d234bcde9fffea80f169e412